src/main/proto/eventtemplates/video_template.proto

```diff
@@ -28,28 +28,25 @@ message Video {
     description: "An Event Template for video ad events."
   };
 
-  // Measures the percentage of a video ad that was viewed.
-  message DigitalVideoCompletionStatus {
-    bool completed_0_percent_plus = 1;
-    bool completed_25_percent_plus = 2;
-    bool completed_50_percent_plus = 3;
-    bool completed_75_percent_plus = 4;
-    bool completed_100_percent = 5;
-  }
+  // These field measure the percentage of a video ad that was viewed.
+  // Should be set cumulatively. e.g. if the completed_50_percent_plus is true
```

Comment:
I'm not going to block on this since my understanding is that alignment was difficult to obtain, but I want to call out that it's poor design to allow invariants to be violated like this. A better design would be either a numerical field representing the completion ratio or an enumeration. Both of those options would make it impossible to have a message where "50%+ completed" is true but "25%+ completed" is false.

Reply:
An enumeration would have the additional benefit of being shareable across templates.